crates/api_models/src/admin.rs

```diff
@@ -66,6 +66,10 @@ pub struct MerchantAccountCreate {
     /// An identifier for the vault used to store payment method information.
     #[schema(example = "locker_abc123")]
     pub locker_id: Option<String>,
+
+    ///Default business details for connector routing
+    #[schema(value_type = Option<PrimaryBusinessDetails>)]
+    pub primary_business_details: Option<serde_json::Value>,
```

Comment:
We are going to support business_details for muliple connector, so we don't need this add for merchant account and other merchant account related structs also

Reply:
This is the default business details for a merchant